app/src/main/java/de/westnordost/streetcomplete/screens/settings/quest_selection/QuestSelectionList.kt

```diff
@@ -88,7 +88,22 @@ fun QuestSelectionList(
                 bottom = contentPadding.calculateBottomPadding()
             ),
         ) {
-            itemsIndexed(reorderableItems, key = { _, it -> it.questType.name }) { index, item ->
+            itemsIndexed(
+                reorderableItems,
+                key = { _, it -> it.questType.name },
+                contentType = { _, _ -> "QuestSelectionItem" },
```

Comment:
this is not necessary, as far as I know, because all items have the same content type (`null` if not specified), it makes no difference here.